content/en/docs/faq/_index.md

```diff
@@ -0,0 +1,12 @@
+---
+title: "FAQ"
+description: "FAQ is designed to answer and summarize the questions our users most frequently ask about KubeSphere."
+layout: "single"
+
+linkTitle: "FAQ"
+weight: 7000
+
+icon: "/images/docs/docs.svg"
+---
+
+FAQ is designed to answer and summarize the questions our users most frequently ask about KubeSphere. This section is in a long-term maintenance for v3.0.x.
```

Comment:
->
FAQ is designed to answer and summarize the questions users ask most frequently about KubeSphere. This section will be updated regularly for v3.0.x.